.vscode/settings.json

```diff
@@ -52,6 +52,7 @@
     "keyfile",
     "ksession",
     "loadercss",
+    "ldot",
```

Comment:
question: what is ldot?

Reply:
lane direction of travel. This was used in variables like "ldotReportData"